src/arrayMethodJoin.js

```diff
@@ -4,9 +4,21 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
```

Comment:
It's better to avoid modifying the native Array prototypes directly. Instead, create a new function and extend the Array.prototype with it.